modules/core/test/v2/unit/coins/xlm.ts

```diff
@@ -468,26 +468,157 @@ describe('XLM:', function() {
       validTransaction.should.equal(true);
     }));
 
-    describe('Federation lookups:', function() {
-      describe('Look up by stellar address:', function() {
-        it('should fail to loop up an invalid stellar address with a bitgo.com domain', co(function *() {
-          const stellarAddress = 'invalid*bitgo.com';
-          try {
-            yield basecoin.federationLookupByName(stellarAddress);
-          } catch (e) {
-            e.message.should.equal('account not found');
-          }
-        }));
+    describe('trustline transactions', function() {
+      it('should fail to verify a trustline transaction with unmatching number of trustlines', co(function *() {
+        const txParams = {
+          recipients: [],
+          type: 'trustline',
+          trustlines: [],
+        };
+
+        const buildResult = {
+          txBase64: 'AAAAANsKrHV2BVjACFt2xlyhxYzP2MNBmb4IQ5E9/WiJiV3TAAABLAAM4aEAAAAHAAAAAAAAAAAAAAABAAAAAAAAAAYAAAABQlNUAAAAAABhNDpbuY4frrgwVQqkws7jxK+k4IMrJ6BaE0OFUva9vwAAAOjUpRAAAAAAAAAAAAA=',
+        };
+
+        nock('https://test.bitgo.com')
+          .post(`/api/v2/${wallet.coin()}/wallet/${wallet.id()}/tx/build`)
+          .reply(200, buildResult);
+
+        const txPrebuild = yield wallet.prebuildTransaction(txParams);
+        const verification = {
+          disableNetworking: true,
+          keychains: {
+            user: { pub: userKeychain.pub },
+            backup: { pub: backupKeychain.pub },
+          },
+        };
+        try {
+          yield basecoin.verifyTransaction({ txParams, txPrebuild, wallet, verification });
```

Comment:
Nit: consider 
```javascript
yield basecoin.verifyTransaction({ txParams, txPrebuild, wallet, verification })
  .should.be.rejectedWith('transaction prebuild does not match expected trustline operations')
```

(this pattern is used several times, this comment applies to all usages)

Reply:
fixed all